ldpd/ldp_vty_conf.c

```diff
@@ -429,6 +431,13 @@ ldp_vty_mpls_ldp(struct vty *vty, const char *negate)
 		vty_conf->flags |= F_LDPD_ENABLED;
 	}
 
+	/* register / de-register to recv info from zebra */
+	bool want_registered = !negate;
+	if (want_registered != zebra_registered) {
+		ldp_zebra_regdereg_zebra_info(want_registered);
+		zebra_registered = want_registered;
```

Comment:
nit: this is unnecessary as `ldp_zebra_regdereg_zebra_info()` is already updating the `zebra_registered` variable (which can probably be declared with static scope).

Reply:
thanks, fixed it